help_to_heat/middleware.py

```diff
@@ -0,0 +1,29 @@
+import logging
+
+from django.core.handlers.wsgi import WSGIRequest
+
+middleware_logger = logging.getLogger(__name__)
+requests_logger = logging.getLogger("requests")
+
+
+class ExceptionMiddleware:
+    def __init__(self, get_response):
+        self.get_response = get_response
+
+    def __call__(self, request):
+        # pass requests down the stack without edits
+        return self.get_response(request)
```

Comment:
Assume this is just standard? Is it necessary?

Reply:
yes, else `'ExceptionMiddleware' object is not callable`